File: web3rpc/rpc-specs/code-samples/javascript/admin/startHTTP.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

    const host = '127.0.0.1'
    const port = 8555
    const cors = ''
    const apis = 'klay'

    provider.admin.startHTTP({ host, port, cors, apis }, (err, data, response) => {
        console.log(data);
    });
}
)()

File: web3rpc/rpc-specs/code-samples/python/admin/startHTTP.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

adminHost = "127.0.0.1"
port = 8551
cors = ""
apis = "klay"

w3 = Web3(Web3.HTTPProvider(host))
admin_response = w3.geth.admin.start_http(adminHost, port, cors, apis)

print(admin_response)
